content/codepipeline/cleanup.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Cleanup"
+date: 2018-10-087T08:30:11-07:00
+weight: 16
+draft: true
+---
+
+Congratulations on making it through the CI/CD with CodePipline module.
+
+This module is not used in subsequent steps, so you can remove the resources now, or at the end of the workshop.
+
+First we need to delete the Kubernetes deployment and service:
+
+```
+kubectl delete deployments hello-k8s
+kubectl delete services hello-k8s
+```
+
+Next, we are going to delete the CloudFormation stack created. Open up CloudFormation the [AWS Managemnt Console](https://console.aws.amazon.com/cloudformation).
```

Comment:
Once we swap to one of the existing workshop apps, I think we don't need to clean up the pipeline until we're cleaning up the whole workshop.